### DISCUSSION TITLE
🚫  Error: PostCSS plugin postcss-nested requires PostCSS 8.

### DISCUSSION THREAD

**Discussion by JimLynchCodes** [tailwindlabs/tailwindcss#3751] [Help] [answered]
### Describe the problem:

I just scaffolded a new project (`create-react-app `) and installed tailwind according to this guide: 
https://www.youtube.com/watch?v=_nRl46Tn1W8&t=611s

However, I'm getting this error:

```
🚫 Error: PostCSS plugin postcss-nested requires PostCSS 8.
Migration guide for end-users:
https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users
```

I tried following this migration guide but can't seem to get this to compile. 

Please halp. 😢

### Link to a minimal reproduction:

the "create-react-tailwind-v1" directory of this repo:
https://github.com/JimLynchCodes/Bid-Ask-Blaster-Bot

using node v14.16.0

thanks!

**Comment by adamwathan** [comment, marked as answer]
Hey! Follow the official instructions, that video is very outdated:

https://tailwindcss.com/docs/guides/create-react-app
